src/PublishCodeCovCoverageTask/__tests__/webUtils.test.ts

```
@@ -0,0 +1,672 @@
+            // Mock successful file handling
+            const fileStreamMock = {
+                on: jest.fn().mockImplementation(function (this: any, event, handler) {
+                    if (event === 'finish') setTimeout(() => handler(), 0);
+                    return this;
+                }),
+                close: jest.fn(cb => cb && cb())
+            };
+
+            // Setup our HTTP get sequence - first call returns redirect, second returns success
+            let callCount = 0;
+            let requestedUrls: string[] = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'requestedUrls' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)